python/django_bolt/urls.py

```
@@ -0,0 +1,116 @@
+
+def _to_django_route(bolt_path: str) -> str:
+    """Convert a Bolt path to a Django route: ``/files/{p:path}`` -> ``files/<path:p>``."""
+
+    def _convert(match: re.Match) -> str:
+        name, converter = match.group(1), match.group(2)
+        return f"<path:{name}>" if converter == "path" else f"<{name}>"
+
+    return _PARAM_RE.sub(_convert, bolt_path).lstrip("/")
+
+
+def _reverse_only_view(*args, **kwargs):  # pragma: no cover - never invoked
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_reverse_only_view`

Add return type annotation: `Never`

(ANN202)